ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -9,8 +9,8 @@
 /* Begin PBXBuildFile section */
```

Comment:
add this file to .gitignore

Reply:
please also remove this file from git

git rm  ios/Runner.xcodeproj/project.pbxproj